inc/class-upgrade.php

```diff
@@ -832,42 +831,14 @@ private function upgrade_165() {
 	}
 
 	/**
-	 * Performs the 16.9 upgrade. shop_order indexables stopped being added in the db since 16.7, so we have to clean out older entries from the indexable table.
+	 * Performs the 17.2 upgrade. Cleans out any unnecessary indexables. See $cleanup_integration->get_cleanup_tasks() to see what will be cleaned out.
 	 *
 	 * @return void
 	 */
-	private function upgrade_169() {
-		$cleanup_integration = YoastSEO()->classes->get( \Yoast\WP\SEO\Integrations\Cleanup_Integration::class );
-		$number_of_deletions = $cleanup_integration->clean_indexables_with_object_type( 'post', 'shop_order', 1000 );
+	private function upgrade_172() {
+		\wp_unschedule_hook( 'wpseo_cleanup_orphaned_indexables' );
```

Comment:
@Xyfi What about https://github.com/Yoast/wordpress-seo/pull/17373#issuecomment-909062012 (unregister the `wpseo_cleanup_indexables` hook too? Is there something I'm missing that makes this unnecessary? 

PS: I see the value now of adding separate comments in the code VS in the PR itself, sorry about that 😁